courses/static/js/course_comparison.js

```diff
@@ -368,6 +375,41 @@ class ComparisonDisplayManager {
 
 }
 
+class InfoBoxManager {
+
+    setup() {
```

Comment:
get that you followed me but this would be better down on the `constructor(){}` then you don't need to call setup. Unless you're delaying the setup for a particular reason

Reply:
Changed the setup to a constructor